src/Ion.ts

```diff
@@ -57,7 +58,7 @@ function makeBinaryReader(span: Span, options: Options) : BinaryReader {
 }
 
 function makeTextReader(span: Span, options: Options) : TextReader {
-  return new TextReader(span, options && options.catalog);
+  return new TextReader(span, options && options.catalog, options && options.raw_tokens);
```

Comment:
We might want to consider factoring out a Lexer and Parser for IonJS and provide typescript interfaces for both. 

Then we can have 2 entry points to the lib 

1. `Ion.ts` for using (read/write) Ion data 
1. `Parser.ts` and `Lexer.ts` that can provide a richer API for tools that would like to reuse/rely on the Ion lexer/parser